### ISSUE TITLE
Copyright assigned to Brightcove

### ISSUE THREAD

**Issue by dmlap** [tim-peterson/videojs-playlist#16]
I was clicking through your repo and noticed LICENSE.md assigns the copyright for this project to Brightcove. Was that intentional?

**Comment by tim-peterson** [comment]
No, sorry I think I just copy pasted it from video.js itself. I have no idea about this copyright stuff and don't particularly care about my own rights so please advise me on what you'd like me to do.

**Comment by dmlap** [comment]
I'm not a lawyer but I think the standard thing to do is assign the copyright to yourself.
